be/src/storage/persistent_index.cpp

```diff
@@ -1800,8 +1800,13 @@ Status ShardByLengthMutableIndex::load(const MutableIndexMetaPB& meta) {
     for (int i = 0; i < n; i++) {
         const auto& page_pointer_pb = meta.wals(i).data();
         auto offset = page_pointer_pb.offset();
-        for (const auto& shard : _shards) {
-            RETURN_IF_ERROR(shard->load(offset, read_file));
+        std::string buff;
```

Comment:
Better to add more concise message to the pull request.

Reply:
ok